tests/helpers/mod.rs

```diff
@@ -238,17 +236,34 @@ impl TestBuilder {
   }
 
   #[allow(unused)]
-  pub fn verify_and_fill_checksums(&mut self, value: bool) -> &mut Self {
-    self.verify_and_fill_checksums = value;
+  pub fn ensure_locker(&mut self) -> &mut Self {
+    self.locker.get_or_insert_with(Default::default);
     self
   }
 
   #[allow(unused)]
```

Comment:
Why these attributes?

Reply:
There are multiple test runners sharing this code and they unfortunately error when one of them doesn't use it.